app/views/foods/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1 class="heading"> All Foods</h1>
+
+<div class="mb-3 col-sm-2 btn-div">
+    <%= link_to "Add food", new_food_path, class:"btn btn-primary form-control" %>
+</div>
+
+<div class="listDiv">
+<table class="table table-striped table-bordered">
+    <thead>
+        <tr>
+        <th >Food</th>
+        <th>Measurement_Unit</th>
+        <th>Unit Price</th>
+        <th>Action</th>
+        </tr>
+    </thead>
+
+    <tbody>
+    <% @foods.each do |food|%>
+    <tr>
+      <td><%= food.name%></th>
+      <td><%= food.measurement_unit%></td>
+      <td>$<%= food.price%></td>
+      <td class="mb-3 col-sm-2">
+      <%if can? :destroy, food%>
+        <%=link_to "Delete", food_path(food.id), method: :delete, class:"btn btn-primary form-control"%>
+      <%end%>
```

Comment:
Please, kindly change `link_to` to `button_to`, because `link_to` does not work with delete method `method: :delete`.
Also, I suggest removing lines `25` and `27` (or comment them) since you didn't install CanCan rails does not recognize `can?`